docs/publishing.md

```diff
@@ -51,7 +51,7 @@ Additional steps for the main line release
   - Mention any security fixes
 - Create Release Notes PR
   - Add the release note file as [`/docs/releases/vx.y.0.md`](./releases)
-  - Add an entry to [`/microsite/sidebar.json`](https://github.com/backstage/backstage/blob/master/microsite/sidebars.json) for the release note
+  - Add an entry to [`/microsite/sidebar.js`](https://github.com/backstage/backstage/blob/master/microsite/sidebars.js) for the release note
```

Comment:
```suggestion
  - Add an entry to [`/microsite/sidebars.js`](https://github.com/backstage/backstage/blob/master/microsite/sidebars.js) for the release note
```

but do we still need to do this?

Reply:
I think the whole point of this is to not have this as a step any longer 😅

Reply:
right? :)